__tests__/integration/CommandsTest.intTest.js

```diff
@@ -0,0 +1,20 @@
+const nbaUtils = require('../../util/nbaUtils')
+const nbaData = require('../../api/nbaData')
+jest.setTimeout(10000)
+
+describe('test suite for /commands/', () => {
+  describe('tests for player.js', () => {
+    test('check that output can handle null', async () => {
```

Comment:
is this test handling null or ensuring that null doesn't get sent back?

Reply:
This test is checking that the return msg should not be null, that said this test might need to be moved under, if it hasn't already, daysPlayedOn()

Reply:
per @Laturine delete this test, unit test already written, and create ticket to write test for a negative case (ie a case in which null gets through daysPlayedOn() )